lumen-php-fpm/Dockerfile

```diff
@@ -47,6 +46,10 @@ RUN docker-php-ext-configure gd \
     --with-png-dir=/usr/include/ \
     --with-jpeg-dir=/usr/include/
 
+RUN apk --no-cache --update --repository http://dl-cdn.alpinelinux.org/alpine/edge/testing/ --repository http://dl-cdn.alpinelinux.org/alpine/edge/main/ add rabbitmq-c rabbitmq-c-dev libressl libressl-dev \
```

Comment:
Split these dependencies up on separate lines using `\` (see other images)

Also, we probably shouldn't use `testing` if possible. Check if packages are available in `community`